projects/ccd-case-ui-toolkit/src/lib/shared/domain/case-file-view/document-tree-node.model.ts

```diff
@@ -4,3 +4,8 @@ export interface DocumentTreeNode {
   count?: number;
```

Comment:
Maybe make type as DocumentTreeNodeType as well?

Reply:
Thank you @MateiHMCTS, I have now made the changes to the code.